src/components/AlchemyRow.vue

```diff
@@ -0,0 +1,176 @@
+<template>
+  <td class="columnSpan">
+    <GameAsset class="centered"
+      :image="Assets.FromDir(`${props.group}${props.idx+1}`, 'alchemy')"
+      :height="64"/>
+  </td>
+  <td class="columnSpan">
+    <input class="columnSpan"
+      type="number" :min="0"
+      :value="alchemy.upgrades[group][props.idx]"
+      @change='customChange'/>
+  </td>
+  <td class="columnSpan">
+    <input class="columnSpan"
+      type="number" :min="alchemy.upgrades[group][props.idx]"
+      :value="alchemy.goals[group][props.idx]"
+      @change="handleAlchemyGoal($event, group, props.idx)"/>
+  </td>
+  <td class="centered columnSpan-2 effect"> 
+    {{effectChange()}}
+  </td>
+  <table class="filled">
+    <tr :style="{display: 'flex'}">
```

Comment:
Try to avoid writing styles like this inline. If it's complex, use a class and write the styles in a `<style>...</style>` at the bottom of the file; otherwise, use [Tailwind](https://tailwindcss.com/) class names.

```suggestion
    <tr class='flex'">
```